src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -62,6 +66,29 @@ public function doSomething(\Foo\Bar $foo): \Foo\Bar\Baz
 }
 '
                 ),
+                new CodeSample(
+                    '<?php
+
+class SomeClass
+{
+    public function doSomething(Foo\Bar $foo)
+    {
+    }
+}
+'
+                ),
+                new CodeSample(
+                    '<?php
+
+class SomeClass
+{
+    public function doSomething(\Foo\Bar $foo)
+    {
+    }
+}
+',
+                    ['no_namespace_backslash' => false]
```

Comment:
@Wirone may I ask for a help with the following assertion:
```
1) PhpCsFixer\Tests\Fixer\Import\FullyQualifiedStrictTypesFixerTest::testFixerDefinitions
[fully_qualified_strict_types] Multiple non-versioned samples with default configuration.
Failed asserting that an array does not have the key 'default'.
```

how to adjust the examples to fix it?

Reply:
@mvorisek there should be only one example with default configuration if fixer implements `ConfigurableFixerInterface`, the other examples should have different config passed or they should be instance of `VersionSpecificCodeSampleInterface`. See `PhpCsFixer\Tests\Test\AbstractFixerTestCase::testFixerDefinitions()`.

Reply:
I believe you're getting error because `FullyQualifiedStrictTypesFixer` currently has 2 code samples without config.. I believe we can remove first one and keep the second, which does the same but additionally has return type.